src/app/views/api/v1/survey.py

```diff
@@ -13,11 +12,12 @@ class SurveyGetFirstPageApiHandler(JsonApiHandler):
     """
     API that gets the first page for a user, somewhat implicitly for now.
     """
-
     def get(self):
         user_id = self.request.GET.get('userId', 1)
-        first_page_to_return = get_first_survey_page_for_user_id(user_id)
-        questions, prev_page, next_page = get_survey_page_for_user_id(first_page_to_return, user_id)
+        quiz_type = self.request.GET.get('quizType', 'fullform')
+
+        survey = survey_for_type(quiz_type)(int(user_id))
```

Comment:
`survey_for_type` returns a Class definition rather than an instantiated object, so that you can then go ahead and instantiate said object using its return.